lib/active_model_serializers/test/serializer.rb

```diff
@@ -0,0 +1,125 @@
+require 'set'
+module ActiveModelSerializers
+  module Test
+    module Serializer
+      extend ActiveSupport::Concern
+
+      included do
+        setup :setup_serialization_subscriptions
+        teardown :teardown_serialization_subscriptions
```

Comment:
We really need this `teardown` ? If we need, I think we need to write a broken test to it, what behavior will be broken if we do not have this `teardown`.